Include/Python-ast.h

```diff
@@ -1,7 +1,15 @@
 /* File automatically generated by Parser/asdl_c.py. */
 
+#ifndef PYTHON_AST_H
+#define PYTHON_AST_H
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Do we need this? This is an internal generated header, it is not purposed for direct use.

Reply:
Yes, it's needed if ast.h is included in file compiled in C++. I know that some people do that. At least, it doesn't hurt to add extern "C" { ... }.